docker_build_dependency_image.sh

```diff
@@ -19,6 +19,8 @@
 # bash docker_build_dependency_image.sh DEVICE={{gpu|tpu}} MODE=stable_stack BASEIMAGE={{JAX_STABLE_STACK BASEIMAGE FROM ARTIFACT REGISTRY}}
 # bash docker_build_dependency_image.sh MODE=nightly
 # bash docker_build_dependency_image.sh MODE=stable JAX_VERSION=0.4.13
+# Nightly build with JAX_VERSION for GPUs. Available versions listed at https://storage.googleapis.com/jax-releases/jax_nightly_releases.html:
```

Comment:
Is there a reason this is GPU specific? I can imagine use cases where this is useful for TPUs as well

Reply:
Yes, the TPU path is installing nightly `jaxlib` using a different command:

```
pip3 install --pre -U jaxlib -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
```

It looks like this command is outdated and versions newer than 0.4.30 don't appear to be available: https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html. We can make the change for TPU also, but we need to change the installation pattern like here: https://jax.readthedocs.io/en/latest/installation.html#jax-nightly-installation. Left this out of this PR to keep the scope down. And also because I needed this on GPUs already since there seemed to be an issue with a nightly build I was using last week

Reply:
Thanks for the extra info, this makes sense!